crates/bevy_asset/src/io/mod.rs

```diff
@@ -72,11 +72,40 @@ impl From<std::io::Error> for AssetReaderError {
     }
 }
 
-pub trait AsyncReadAndSeek: AsyncRead + AsyncSeek {}
+pub const STACK_FUTURE_SIZE: usize = 10 * std::mem::size_of::<&()>();
```

Comment:
Doc string here, including a brief justification of about why that number was chosen.